jumpscale/packages/vdc/services/check_threebot.py

```diff
@@ -0,0 +1,85 @@
+from jumpscale.loader import j
+
+from jumpscale.sals.zos import get as get_zos
+from jumpscale.tools.servicemanager.servicemanager import BackgroundService
+from jumpscale.clients.explorer.models import WorkloadType
+
+
+class CheckThreebot(BackgroundService):
```

Comment:
let's make that runs hourly

Reply:
It already runs hourly, interval defined in line 9